student.rb

```diff
@@ -1,4 +1,4 @@
-require './classroom'
+require_relative './classroom'
 class Student < Person
```

Comment:
- [ ] The Person dependency is missing in this class. It actually prevents the app from running. Here is the error
```
uninitialized constant Person (NameError)

class Student < Person
```
It would work if you also required the Person class here. Let's fix it